src/actors/FilterBarActor.js

```diff
@@ -0,0 +1,88 @@
+import * as SharedUtils from '../utils/SharedUtils';
+
+export class FilterBarActor {
```

Comment:
The examples I've seen use events, but they still have to pass the ActionCreator to the component, or declare it as a singleton that is accessible to any instances on the page.

I'd love to have a button._onClick event be something like

```Javascript
publishEvent('applyButtonClicked')
```

But I'm not sure how to achieve that without a global messaging system?